src/App.tsx

```diff
@@ -20,204 +69,30 @@ export const App: React.FC = () => {
 
       <div className="todoapp__content">
         <header className="todoapp__header">
-          <button
-            data-cy="ToggleAllButton"
-            type="button"
-            className="todoapp__toggle-all active"
+          <NewTodo
+            todos={userTodos}
+            hasTodos={hasTodos}
+            handleLoadTodos={handleLoadTodos}
+            query={query}
+            setQuery={setQuery}
+            setIsEditing={setIsEditing}
```

Comment:
Denys tell that it is bad practise to forward in props setState, let`s create the handlers and forward them in props